components/form/UploadImageInput.tsx

```diff
@@ -72,7 +79,7 @@ const UploadImageInput: React.FC<UploadImageProps> = ({
         type='file'
         name='image'
         onChange={(e) => imageFileUpload(e)}
-        required
+        required={isImageRequired}
```

Comment:
I like your attempted at making this component more reusable but we can add some clarity.

Check our documentation for [guidance on using JS Docs](https://github.com/enBloc-org/kindly/blob/dev/.github/BEST_PRACTICE.md#documenting-through-js-docs) which could be very useful here to explain to other developers the appropriate application of your optional prop. 


I would also consider naming it differently. For example, this prop could be name _isRequired_ to indicate it's value is a boolean and it could assume a value of _true_